### PR TITLE
Fix up typo in felix.py

### PR DESCRIPTION
**Pull request by plwhite**
No tests of this code - because it's not clear that testing this is higher priority than filling in other holes in our testing and coverage.

Fixes bug #176.

### CODE REVIEW

File: calico/felix/felix.py [hunk 1]
```diff
@@ -547,21 +547,27 @@ def _update_endpoint(self, endpoint, fields):
...
+            log.error("No state for endpoint %s : %s",
+                      endpoint.uuid,
+                      fields)
+            raise InvalidRequest("No state for endpoint %s" % endpoint.uuid)
 
         if state not in Endpoint.STATES:
-            log.error("Invalid state %s for endpoint %s : %s" %
-                      (state, endpoint.uuid, state))
+            log.error("Invalid state %s for endpoint %s : %s",
+                      state, endpoint.uuid, fields)
```
Comment (Lukasa) [inline calico/felix/felix.py]: Is this change right?

Comment (plwhite) [inline calico/felix/felix.py]: Yes, I think so.

We are trying to report the invalid state, the endpoint it applied to and the entire request. Not guaranteeing that it's really not got some obvious dumb typo that I just cannot see, but it looks right.

**Comment by Lukasa** [comment]
retest this please

**Comment by Lukasa** [comment]
retest this please

**Pull request merged**